build/package/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:1.18
+
+RUN git -c http.sslVerify=false clone https://github.com/RedHatCRE/syncron.git
+WORKDIR /go/syncron/
+RUN touch syncron.yaml
+RUN go mod tidy
+RUN go install cmd/adhoc/main.go
```

Comment:
FYI This command will install the binary under: '~/go/bin' as 'adhoc' without executable rights. You will probably need to both rename it and apply some chmod to it. That is at least what I had to do on my machine, Docker may be different.